gtm/README.md

```diff
@@ -0,0 +1,141 @@
+# Search Insights for Google Tag Manager
+
+Google Tag Manager [custom template](https://developers.google.com/tag-manager/templates/) for Algolia Search Insights.
+
+---
+
+<div align="center">
+
+[**Download template**](generated/search-insights.tpl)
+
+</div>
+
+---
+
+## Usage
+
+### 1. Import the template
+
+- Download **[`search-insights.tpl`](generated/search-insights.tpl)**
+- Import the file in [Google Tag Manager](https://tagmanager.google.com) (GTM)
+  - Go to "Templates"
+  - Click on "New"
+  - Select "Import" in the menu
+
+### 2. Create variables
+
+For events to be sent with the correct information (`userToken`, `objectIDs`, `queryID`, etc.), you need to add variables in the "User-Defined Variables" section.
+
+[Learn more about GTM variables →](https://www.simoahava.com/analytics/variable-guide-google-tag-manager/)
+
+### 3. Create tags
+
+1. Click "New" in the Tags panel
+1. Select the tag type "Algolia Search Insights"
+1. Choose the event type and provide the Search Insights options
+1. Select the trigger for this event
+
+## FAQ
+
+### How to get the user token?
+
+The user token can be given to GTM via a [Data Layer](https://developers.google.com/tag-manager/devguide#datalayer).
+
+```js
+window.dataLayer = window.dataLayer || [];
+window.dataLayer.push({
+  userToken: 'USER_1',
+});
+```
+
+### How to get the object IDs?
```

Comment:
Do we need a similar one for `position`?

Reply:
I agree it's might be worth documenting positions as well.
Actually since the steps are exactly the same for `objectID`, `position` and `queryID` maybe we can group them together and give the user a single snippet to copy-paste?

Reply:
That's right, I'll add a part on `__position`.